.idea/mindful-breathing-pt2.iml

```diff
@@ -2,7 +2,7 @@
 <module type="PYTHON_MODULE" version="4">
   <component name="NewModuleRootManager">
     <content url="file://$MODULE_DIR$" />
-    <orderEntry type="inheritedJdk" />
+    <orderEntry type="jdk" jdkName="Python 3.6" jdkType="Python SDK" />
```

Comment:
We use the 3.5+ version of Python, not a big difference but best to keep this file as it is just in case changes could cause problems